Header header
float32 drone_timeuint16 tag
uint16 size
uint32 version
uint8[] data_frame
